src/bernoulli.jl

```diff
@@ -0,0 +1,118 @@
+#
+# Bernoulli numbers (as rationals up to B_{34}) and Bernoulli polynomials
+# The alg. for the latter could use some optimisation, as
+#    (1) the choice of domains for the different approaches was a bit arbitrary
+#    (2) the errors start to creep up for larger n, looks roughly like 1 order per increase in n of 5
+#
+
+"""
+    bernoulli(n)
+
+ created: 	Tue May 23 2017 
```

Comment:
We typically don't include this information in docstrings, as its readily available in the Git history.